prov/verbs/src/ep_rdm/verbs_queuing.h

```diff
@@ -204,6 +204,34 @@ fi_ibv_rdm_move_to_postponed_queue(struct fi_ibv_rdm_request *request)
 }
 
 static inline void
+fi_ibv_rdm_remove_from_multi_recv_list(struct fi_ibv_rdm_multi_request *request,
+				       struct fi_ibv_rdm_ep *ep)
+{
+	dlist_remove(&request->list_entry);
+}
+
+static inline void
+fi_ibv_rdm_add_to_multi_recv_list(struct fi_ibv_rdm_multi_request *request,
+				  struct fi_ibv_rdm_ep *ep)
+{
+	dlist_insert_tail(&request->list_entry,
+			  &ep->fi_ibv_rdm_multi_recv_list);
+}
```

Comment:
The above 2 functions are needless abstractions.  Not blocking this PR because of that, however.